src/audio/Kconfig

```diff
@@ -6,4 +6,10 @@ config VOLUME
 	help
 	  Select for Volume component
 
+config SRC
+	bool "SRC"
```

Comment:
Same comment here, this sould be called COMP_SRC

Reply:
Changed to COMP_SRC